jbpm-wb-case-mgmt/jbpm-wb-case-mgmt-client/src/main/java/org/jbpm/workbench/cm/client/roles/CaseRolesPresenter.java

```diff
@@ -48,42 +56,150 @@ public String getTittle() {
 
     @Override
     protected void clearCaseInstance() {
-        view.disableNewRoleAssignments();
         view.removeAllRoles();
     }
 
     @Override
     protected void loadCaseInstance(final CaseInstanceSummary cis) {
-        view.addUser(cis.getOwner(), translationService.format(OWNER));
-
-        setupRoleAssignments(cis);
-
-        setupNewRoleAssignments(cis);
-
-        view.setupPagination();
+        if (passFilter(Arrays.asList(cis.getOwner()), null)) {
+            view.addOwnerAssignment(translationService.format(OWNER), cis.getOwner());
+        }
```

Comment:
Not sure the evaluation of cis.getOwner() has a value here. The case instance can't be created without providing the owner, which means that if condition will always evaluate to _true_.